src/python/pants/option/global_options.py

```diff
@@ -76,7 +78,8 @@ def from_bootstrap_options(cls, bootstrap_options):
     remote_store_chunk_bytes=1024*1024,
     remote_store_chunk_upload_timeout_seconds=60,
     remote_store_rpc_retries=2,
-    process_execution_parallelism=multiprocessing.cpu_count()*2,
```

Comment:
Leaving this at `*2` would be good probably. The reason being that work execute here is unlikely to be CPU-bound, and _most_ of the tools we invoke are not well multi-threaded.

Reply:
This default was never actually being used before, but happy to change it "back". https://github.com/pantsbuild/pants/pull/7960/files/df3b70f96a2879088facc6e3abe8bbf561c08f55#diff-75fd6e699e8dc92731600eccad4e1520L391